Makefile

```diff
@@ -142,52 +76,41 @@ endif
 
 all: install lint test
 
-build: BUILD_ARGS=-o $(BUILDDIR)/
-
-build: go.sum $(BUILDDIR)/ dbbackend
-	CGO_CFLAGS=$(CGO_CFLAGS) CGO_LDFLAGS=$(CGO_LDFLAGS) CGO_ENABLED=$(CGO_ENABLED) go build -mod=readonly $(BUILD_FLAGS) $(BUILD_ARGS) ./...
+$(TEMPDIR)/:
+	mkdir -p $(TEMPDIR)/
+
+wasmvmlib: $(TEMPDIR)/
+	@mkdir -p $(TEMPDIR)/lib
+    ifeq (",$(wildcard $(TEMPDIR)/lib/libwasmvm*.a)")
+        ifeq ($(OS_NAME),darwin)
+	        wget https://github.com/Finschia/wasmvm/releases/download/$(WASMVM_VERSION)/libwasmvmstatic_darwin.a -O $(TEMPDIR)/lib/libwasmvmstatic_darwin.a
+        else
+            ifeq ($(ARCH),amd64)
+	            wget https://github.com/Finschia/wasmvm/releases/download/$(WASMVM_VERSION)/libwasmvm_muslc.x86_64.a -O $(TEMPDIR)/lib/libwasmvm_muslc.a
+            else
+	            wget https://github.com/Finschia/wasmvm/releases/download/$(WASMVM_VERSION)/libwasmvm_muslc.aarch64.a -O $(TEMPDIR)/lib/libwasmvm_muslc.a
+            endif
+        endif
+    endif
 
-install: go.sum $(BUILDDIR)/ dbbackend
-	CGO_CFLAGS=$(CGO_CFLAGS) CGO_LDFLAGS=$(CGO_LDFLAGS) CGO_ENABLED=$(CGO_ENABLED) go install $(BUILD_FLAGS) $(BUILD_ARGS) ./cmd/fnsad
+# command for make build and make install
+build: BUILDARGS=-o $(BUILDDIR)/
+build install: go.sum $(BUILDDIR)/ wasmvmlib
```

Comment:
What is the reason for the `build install`? Instead of `install`.

Reply:
No, in this case, build and install command are same. So I used `$@`.

Reply:
Understood, thank you.